src/content/advanced/cost optimization/index.md

```diff
@@ -0,0 +1,49 @@
+---
+linkTitle: Cost-optimization
+title: Cost optimization tools
+description: Cost is always a concern when running IT infrasturcture. This document is a curated list of cost optimization tools that we have found to be valuable for our customers and oursleves.
+weight: 100
+menu:
+  main:
+    parent: advanced
+user_questions:
+- What cost optimization tools do you reccomend?
+- How can I tune my setup to optimize for cost?
+aliases:
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+---
+
+# Cost optimization in Kubernetes
+
+Adoption of Kuberenetes still requires monitoring cost drivers. Improving resource utilization allows you to get the most out of your infrastructure, while maintaining your budget. This document is comrpised of a curated list of helpful tools on the journey to cost optimization.
```

Comment:
If we mention that this list is in parts based on our pwn experience, plus the experience of our customers, this would help to highlight the relevance of this list and the special position we are in.